uber/model_checks.py

```diff
@@ -190,6 +190,27 @@ def printed_badge_deadline(attendee):
 
 
 @validation.Attendee
+def printed_badge_change(attendee):
+    badge_name_changes_allowed = True
+
+    # this is getting kinda messy and we probably need to rework the entire concept of "printed badge deadline".
```

Comment:
I don't agree with this in that I don't think this is particularly messy, though I think you're right in that we now effectively have two custom name deadlines - one for preassigned badges and one for swag.

By the way, if you want this to look cleaner, you could always just copy the code I put in the regform template that makes the field read only.